bundles/org.openhab.ui/web/src/components/cards/equipment-card.vue

```diff
@@ -24,42 +24,21 @@
 
 <script>
 import mixin from '@/components/widgets/widget-mixin'
-import itemDefaultListComponent from '@/components/widgets/standard/list/default-list-item'
+import itemDefaultListComponent, { equipmentListComponent } from '@/components/widgets/standard/list/default-list-item'
```

Comment:
It seems `itemDefaultListComponent` is not used anymore so you can remove it from this import.